requirements/requirements.in

```diff
@@ -37,3 +37,7 @@ web3
 typing_extensions
 objgraph
 ulid-py
+jaeger-client==4.8.0
+dbapi-opentracing==0.0.5
+Flask-Opentracing==1.1.0
+requests-opentracing==0.3.0
```

Comment:
Versions shouldn't generally be pinned in the `.in` files (unless there's a very specific reason) as that breaks automatic upgrades (e.g. via dependabot).